### PR TITLE
Update notebooks for echopype 0.6.0

### PR DESCRIPTION
**Pull request by emiliom**
I've gone through all 3 notebooks.
- echopype-tour now works fully. I just have some markdown cleanups to do.
- The 2nd notebook (Pacific hake survey) also works. But I have to do a lot of markdown cleanup.
- For the 3rd notebook (OOI), I get a kernel death on the combine_echodata operation. @lsetiawan maybe you can tackle this one since you're the most familiar with this notebook? Once you get past that step, and after creating `ds_MVBS` you'll need to modify that dataset to deal with the new channel vs frequency_nominal structure. See what I do in the echopype-tour notebook. Basically, it looks like this:
  ```python
  ds_MVBS = (
      ds_MVBS
      .set_coords('frequency_nominal')
      .swap_dims({"channel": "frequency_nominal"})
      .reset_coords('channel')
  )
  ```
  (Of course, this only works when frequencies are not repeated, but that's not a problem in these notebooks)

Addresses https://github.com/OSOceanAcoustics/echopype/issues/663

### CODE REVIEW

File: binder/environment.yml [hunk 1]
```diff
@@ -4,8 +4,8 @@ channels:
 dependencies:
   - python=3.9
   - ipykernel
-  - echopype=0.5.4
-  - s3fs=2021.10.1
+  # - echopype=0.5.4
```
Comment (lsetiawan) [inline binder/environment.yml:7]: Should this be specified? `0.6.0` is not released yet, but would be good to pin the echopype?

Comment (emiliom) [reply binder/environment.yml:7]: Sorry, I haven't updated the environment file. I built my local env using pip install -e . So, yes, in the final version, it should be specified to 0.6.0. For s3fs I have `2022.3.0`, but I don't remember if I pinned it.

**Comment by emiliom** [comment]
Thanks @lsetiawan for running and fixing the OOI notebook! The only thing I'd like to see changed now is this line at the start of section 3.8.4:
```python
ds_Sv = ds_Sv.set_coords('frequency_nominal').swap_dims({"channel": "frequency_nominal"})
```
For consistency with the `ds_MVBS` statement later in the notebook, and also to get the `frequency_nominal` value to display as the plot title, please replace that statement with:  
```python
  ds_Sv = (
      ds_Sv
      .set_coords('frequency_nominal')
      .swap_dims({"channel": "frequency_nominal"})
      .reset_coords('channel')
  )
  ```
Now, if you want to get carried away a bit, feel free to turn those steps into a little function that then gets used twice :smile_cat:

**Comment by emiliom** [comment]
> Will merge once we have echopype v0.6.0 on conda -forge.

See #25. Once 0.6.0 is on conda-forge, I'll take those actions, push the commits to this PR and if everything looks good we can merge it then.

**Comment by leewujung** [comment]
Sounds good. I'll let you do the merge then!

**Comment by emiliom** [comment]
Updated and looking good! Merging.

**Pull request merged**